geopandas/tools/clip.py

```diff
@@ -65,14 +82,22 @@ def clip(gdf, mask, keep_geom_type=False):
     If there are multiple polygons in mask, data from `gdf` will be
     clipped to the total boundary of all polygons in mask.
 
+    If the `mask` is a tuple of `(minx, miny, maxx, maxy)`, a faster rectangle
```

Comment:
```suggestion
    If the ``mask`` is a tuple of ``(minx, miny, maxx, maxy)``, a faster rectangle
```
ReStructuredText needs double backticks, otherwise it treats it as italic.

Reply:
Seeing the rest of the docstring, can you also add them at other (existing) places?

Reply:
Done